scenarios/kubernetes_e2e.py

```diff
@@ -364,6 +367,7 @@ def set_up_kops_aws(workspace, args, mode, cluster, runner_args):
         '--kops-nodes=%s' % args.kops_nodes,
         '--kops-ssh-key=%s' % aws_ssh,
         "--kops-ssh-user=admin",
+        "--kops-multiple-zones=%s" args.kops_multiple_zones,)
```

Comment:
don't append if the flag is not set?

Reply:
It is set with a default in when I added the arg ...